pylintrc_tweaks

```diff
@@ -15,6 +15,7 @@ disable+ =
     useless-suppression,
     cyclic-import,
     logging-format-interpolation,
+    F0001,
```

Comment:
Why was this added?  Ignoring fatal errors seems pretty drastic.

Reply:
This was added in order to handle the only remaining pylint error [no module named sandbox-packages](https://build.testeng.edx.org/job/edx-platform-quality-pipeline-master/6383/pylint/). 
I couldn't find any fix for the error so I disabled this instead.

Reply:
This looks like it happens because when [top_python_dirs()](https://github.com/edx/edx-platform/blob/master/pavelib/quality.py#L57-L74) is given "common" as an argument, it adds "sandbox-packages" to the list of directories/packages to run pylint on, but "pylint sandbox-packages" chokes with that error.  I suspect the real problem here is the hyphen in the package name, but a short term fix would be to exclude that name from the returned list since it doesn't work anyway; we can create a separate ticket for being able to run pylint on the code in that directory.

It looks like "\_\_pycache\_\_" is also included in the list of directories to run pylint on; we should probably exclude that also, as it's useless at best and might even be hurting performance.

Reply:
I'd tried changing the module name to Camel Case notation and removing hyphen from its name but it still generated same error so that's probably not the reason for this error. 
For now, excluding the module seems like a nice idea so I'll implement that instead and create a backlog issue to able running pylint on this module.